app/scripts/dimApp.config.js

```diff
@@ -82,7 +82,7 @@
         //Track Our Initial Activity of Starting the App
         $rootScope.trackActivity();
 
-        console.log('DIM v3.7.3 - Please report any errors to https://www.reddit.com/r/destinyitemmanager');
+        console.log('DIM v$DIM_VERSION - Please report any errors to https://www.reddit.com/r/destinyitemmanager');
         dimInfoService.show('20160603v370', {
           title: 'DIM v3.7.3 Released',
           view: 'views/changelog-toaster.html?v=v3.7.3',
```

Comment:
We could, but since these are sort of "release notes" I don't know if they need to be more manual.